code/index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width , initial-scale=1">
+
+<head>
+  <script src="script-math.js"></script>
+  <script src="script-else.js"></script>
+  <link rel="stylesheet" id="pagestyle" type="text/css" href="theme-default.css">
+  <title>Calculator</title>
+</head>
+
+<body>
+  <h1>Calculator by Roman</h1>
+  <div id="id-container">
+    <form name="nameCalculator" id="id-calculator">
+      <table>
+        <tr>
+          <td colspan="3"><input type="text" name="nameDisplayMain" id="id-displayMain" maxlength="1000" disabled></td>
+          <td><input type="button" name="nameButtonClear" id="id-buttonClear" value="C" onclick="clearDisplay();"></td>
```

Comment:
id and classes not with camel 
so ... id-button-clear

Reply:
no camelCase for id and class in whole code... will be commited once more fixix are done